sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-obo/README.md

```diff
@@ -41,17 +37,13 @@ We need to ensure that this [environment checklist][ready-to-run-checklist] is c
       <artifactId>spring-security-oauth2-resource-server</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.springframework.security</groupId>
-      <artifactId>spring-security-oauth2-client</artifactId>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-oauth2-client</artifactId>
```

Comment:
1. Can we delete `spring-boot-starter-security`?
2. Can we delete `msal4j`?
3. Please sort the dependencies, and add comments in pom file to make it easier to read. Like this: https://github.com/chenrujun/azure-sdk-for-java/blob/3e66ff7ec96e8d695b212a1f992c322cfe01c612/sdk/spring/azure-spring-boot-test-aad-obo/pom.xml

Reply:
get it